jumpscale/packages/tfgrid_solutions/chats/exposed.py

```diff
@@ -234,7 +236,7 @@ def reservation(self):
 
         trc_log_config = j.core.config.get("LOGGING_SINK", {})
         if trc_log_config:
-            trc_log_config["channel_name"] = self.solution_name + "-trc"
+            trc_log_config["channel_name"] = f"{self.threebot_name}-{self.solution_name}-trc"
```

Comment:
I think we can do `.replace("_", "-")`  for 3bot name too because some 3Bots have underscores can lead to invalid domain, we already check the solution name to be a valid identifier

Reply:
then it would conflict if there is the same threebot name with -? 
like if we have both threebot foo_bar and foo-bar?

Reply:
but underscores is not acceptable, no ? maybe we can allow override in that case